app/src/main/java/com/asfoundation/wallet/promotions/ui/from_notification/AlmostVipNavigator.kt

```diff
@@ -0,0 +1,40 @@
+package com.asfoundation.wallet.promotions.ui.from_notification
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import by.kirich1409.viewbindingdelegate.viewBinding
+import com.asf.wallet.R
+import com.asf.wallet.databinding.AlmostVipFragmentBinding
+import com.asfoundation.wallet.base.SingleStateFragment
+import com.asfoundation.wallet.promotions.ui.PromotionsSideEffect
+import com.asfoundation.wallet.promotions.ui.PromotionsState
+import com.asfoundation.wallet.viewmodel.BasePageViewFragment
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class AlmostVipNavigator : BasePageViewFragment(),
```

Comment:
Called `...Navigator` but actually is a Fragment. And it duplicates the `AlmostVipFragment`
Is it even used for something?
If not, remove please